netlify/functions/movie.ts

```diff
@@ -0,0 +1,62 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+import axios from "axios";
+import {
+    HandlerEvent,
+    HandlerContext,
+    HandlerResponse,
+  } from "@netlify/functions";
+import { Path } from "path-parser";
+import { ok } from "./utils/responses";
+import { methodNotAllowed } from "./utils/responses";
+import { StringRecord } from "./utils/types";
+
+export const path = new Path<StringRecord>("/api/movie/:movieId<\\d+>");
+
+const tmdbApiKey = process.env.TMDB_API_KEY
+
+/**
+ * 
+ * GET /movie/:movieId -> returns full movie data
+ * 
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+//exports.handler = async function(event, _context) {
+export async function handler(
+    event: HandlerEvent,
+    context: HandlerContext
+    ): Promise<HandlerResponse> {
+        if (event.httpMethod === 'GET') {
+            const pathArray = event.path.split("/");
+            const movieIndex = pathArray.indexOf('movie') + 1;
+            const movieId = parseInt(pathArray[movieIndex]);
+            
+            if (isNaN(movieId)) {
+                return {
+                    statusCode: 422,
```

Comment:
The club endpoint uses a 404 not found response for invalid clubId, I think we should do the same here